pom.xml

```diff
@@ -2762,8 +2762,10 @@
           <failOnViolation>true</failOnViolation>
           <includeTestSourceDirectory>false</includeTestSourceDirectory>
           <failOnWarning>false</failOnWarning>
-          <sourceDirectory>${basedir}/src/main/scala</sourceDirectory>
-          <testSourceDirectory>${basedir}/src/test/scala</testSourceDirectory>
+          <sourceDirectories>${basedir}/src/main/scala
+            <dir>${basedir}/src/main/scala</dir>
+            <dir>${basedir}/src/test/scala</dir>
+          </sourceDirectories>
```

Comment:
With `sourceDirectories` (http://www.scalastyle.org/maven.html), we can avoid skipping `test` source code check.